newIDE/app/src/ProjectManager/ExtensionsPropertiesDialog.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+import { Trans } from '@lingui/macro';
+
+import * as React from 'react';
+import FlatButton from '../UI/FlatButton';
+import Dialog from '../UI/Dialog';
+import Text from '../UI/Text';
+import { ColumnStackLayout } from '../UI/Layout';
+import EmptyMessage from '../UI/EmptyMessage';
+import PropertiesEditor from '../PropertiesEditor';
+import propertiesMapToSchema from '../PropertiesEditor/PropertiesMapToSchema';
+import { List } from '@material-ui/core';
+
+type Props = {|
+  project: gdProject,
+  open: boolean,
+  onApply: Function,
+|};
+
+class ExtensionsPropertiesDialog extends React.Component<Props> {
+  _onApply = () => {
+    this.props.onApply();
+  };
```

Comment:
Uber nit: no need to create an intermediate function, just pass `this.props.onApply`.